examples/virtual-dom-wip/platform/Html/Internal/Shared.roc

```diff
@@ -43,7 +43,7 @@ Attribute state : [
 
 CyclicStructureAccessor : [
     ObjectField Str CyclicStructureAccessor,
-    ArrayIndex Nat CyclicStructureAccessor,
+    ArrayIndex U64 CyclicStructureAccessor,
```

Comment:
I'd really prefer u32 since this is meant to be sent from wasm to JS and that doesn't play well with 64 bit

Reply:
Addressed in [f83ebb7](https://github.com/roc-lang/roc/pull/4974/commits/f83ebb7bc75c5d6b684eb89fb6b14eff65b5e929)